cubeseed_login/package.json

```diff
@@ -14,6 +14,7 @@
     "next": "13.3.0",
     "react": "18.2.0",
     "react-dom": "18.2.0",
+    "react-responsive-modal": "^6.4.2",
```

Comment:
Please consult with the tech stack here https://docs.google.com/document/d/1otSVVp7G5-K1eNmDUV5-jHWlj8JHG9KH3xgwZGicbBg/edit before adding a new library.

Reply:
Did you make changes to all 80 files?

Reply:
no ..the last person did not commit there work thats why

Reply:
i have check the tech  stack ...cant i use any other library?